### PR TITLE
propagate changes from #525 to other librispeech recipes

### PR DESCRIPTION
**Pull request by marcoyang1998**
Propagate changes made by @csukuangfj in [https://github.com/k2-fsa/icefall/pull/525](url) to other LibriSpeech recipes. It filters out infinite numbers during loss computation.

### CODE REVIEW

File: egs/librispeech/ASR/pruned_transducer_stateless/train.py [hunk 1]
```diff
@@ -544,14 +544,47 @@ def compute_loss(
...
+            logging.info(
+                "Not all losses are finite!\n"
+                f"simple_loss: {simple_loss}\n"
+                f"pruned_loss: {pruned_loss}"
+            )
+            display_and_save_batch(batch, params=params, sp=sp)
+            simple_loss = simple_loss[simple_loss_is_finite]
+            pruned_loss = pruned_loss[pruned_loss_is_finite]
+
+            # If the batch contains more than 10 utterance AND
```
Comment (csukuangfj) [inline egs/librispeech/ASR/pruned_transducer_stateless/train.py:562]:
```suggestion
            # If the batch contains more than 10 utterances AND
```

Comment (marcoyang1998) [reply egs/librispeech/ASR/pruned_transducer_stateless/train.py]: Fixed

File: egs/librispeech/ASR/pruned_transducer_stateless/model.py [hunk 2]
```diff
@@ -95,6 +96,7 @@ def forward(
               lm_scale * lm_probs + am_scale * am_probs +
```
Comment (csukuangfj) [inline egs/librispeech/ASR/pruned_transducer_stateless/model.py:96]: Could you also update the argument documentation about `reduction`?

**Comment by csukuangfj** [comment]
Thanks!
